lib/ruby_lsp/server.rb

```diff
@@ -265,6 +265,19 @@ def run_initialize(message)
                     ],
                   ),
                 ),
+                # Register watching .rubocop.yml
+                Interface::Registration.new(
+                  id: "workspace/didChangeWatchedFiles-rubocop",
+                  method: "workspace/didChangeWatchedFiles",
+                  register_options: Interface::DidChangeWatchedFilesRegistrationOptions.new(
+                    watchers: [
+                      Interface::FileSystemWatcher.new(
+                        glob_pattern: "**/.rubocop.yml",
```

Comment:
Since this is now server-side, I wonder if we could hook into RuboCop to know which files to watch 🤔 .